packages/docs/render-blocks.md

```diff
@@ -0,0 +1,109 @@
+# Render Blocks
+
+In block suite, blocks can be rendered by any UI framework.
+A block should be rendered to a DOM element, and we use `view` to represent the renderer.
+
+By default, we provide a [lit](https://lit.dev/) renderer called `@blocksuite/lit`.
+But it's still possible to use other UI frameworks.
+I'll introduce later about how to write a custom renderer in [custom renderer](/#WIP).
```

Comment:
Stick to using "we" should be the best practice.

Reply:
Done